projects/plugins/crm/readme.txt

```diff
@@ -294,13 +294,13 @@ For more documentation, please see the CRM knowledge base: [https://kb.jetpackcr
 
 == Frequently Asked Questions ==
 
-= Is there any limits to customer or invoice counts in Jetpack CRM? =
+= Is there any limits to contact or invoice counts in Jetpack CRM? =
 
 There are absolutely no limits in Jetpack CRM, apart from the usual ones applied by your host (database size etc.), you can create as many clients or invoices as you'd like!
 
 = Is Jetpack CRM really free? = 
 
-Totally. The core is a solid, useful, Customer Relationship Manager right out of the box. You don't need anything else to get started managing your leads, and customers. We do build extensions, (because we need them), but they're not at all essential. Further, they're cheap in comparison to the pay-per-month CRM options.
+Totally. The core is a solid, useful, Contact Relationship Manager right out of the box. You don't need anything else to get started managing your leads, and contacts. We do build extensions, (because we need them), but they're not at all essential. Further, they're cheap in comparison to the pay-per-month CRM options.
```

Comment:
Ironically enough, then CRM does stand for `Customer Relationship Management`, so we should probably stick with that terminology to help with SEO etc 😄

Reply:
Fixed that now, thanks for catching and mentioning that!